moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -246,8 +246,14 @@ export const useColumns = ({ hiddenColumns }) => {
         headerName: "Modified",
         description: "Date record was last modified",
         field: "updated_at",
-        valueFormatter: (value) => formatTimeStampTZType(value),
         width: COLUMN_WIDTHS.small,
```

Comment:
this wasn't in the issue, but i'm now noticing that we could update this to `COLUMN_WIDTHS.medium` to keep the secondary part of the timestamp from wrapping in cell when its default width loads. Could you make that change? Thanks!